content/en/platform/corda/5.0/developing-applications/api/api-crypto.md

```diff
@@ -101,4 +97,25 @@ dependencies {
 You must reference `net.corda:corda-crypto-extensions`.
 {{< /note >}}
 
-<!--For information about packaging a CorDapp, see the [development tutorial]().-->
\ No newline at end of file
+## Implementing Signature Schemes
+
+Corda supports the following `SignatureSpecs` (signature schemes) for creating `java.security.Signature` objects:
+* SHA256withRSA
+* SHA384withRSA
+* SHA512withRSA
+* RSASSA-PSS with SHA256
+* RSASSA-PSS with SHA384
+* RSASSA-PSS with SHA512
+* RSASSA-PSS with SHA256 and MGF1
+* RSASSA-PSS with SHA384 and MGF1
+* RSASSA-PSS with SHA512 and MGF1
+* SHA256withECDSA
+* SHA384withECDSA
+* SHA512withECDSA
+* EdDSA
+* SHA512withSPHINCS256
+* SM3withSM2
+* SHA256withSM2
+* GOST3411withGOST3410
+
+Use `SignatureSpecService` to retrieve the `SignatureSpec` implemented.
```

Comment:
I think 'Use `SignatureSpecService` to retrieve implemented `SignatureSpec`.' sounds a bit better?
Or even 'Use `SignatureSpecService` to retrieve the `SignatureSpec`.'